weatherman.py

```diff
@@ -0,0 +1,61 @@
+import argparse
+from code_files.weather_parser import WeatherParser
+from code_files.weather_reporter import WeatherReporter
+from code_files.report_calculator import ReportCalculator
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser()
+    parser.add_argument("extract_to", type=str)
+    parser.add_argument("-e", "--extremes", type=int)
+    parser.add_argument("-a", "--averages", type=str)
+    parser.add_argument("-c", "--chart", type=str)
```

Comment:
Can we use `datetime` type instead of `int` here

Reply:
[Using datetime](https://github.com/ttqureshi/WeatherMan/pull/1/commits/b1f61ff2496d9efa085df59157f39ad21df4b856)